LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/MetricRegistry.cs

```diff
@@ -109,24 +116,42 @@ internal static class MetricExporterHelper
         /// Gets the tags ordered by the original order of the tags.
         /// Falls back to common tags that are present in the RegistryMetricTagBag.
         /// </summary>
-        public static string[] GetTagsInOrder(IEnumerable<string> tagNames, ReadOnlySpan<KeyValuePair<string, object?>> tags, RegistryMetricTagBag metricTagBag)
+        public static string[] GetTagsInOrder(IReadOnlyList<string> tagNames, ReadOnlySpan<KeyValuePair<string, object?>> tags, RegistryMetricTagBag metricTagBag)
```

Comment:
if we provide more values than initially dimensions specified, wouldn't we simply drop the value?

Reply:
Done in 8a74c1eb66044ec1ea885e1cb2568aad5e5e32bf